schema.sql

```diff
@@ -2,4 +2,12 @@
 
 CREATE TABLE animals (
     name varchar(100)
-);
+    
+CREATE TABLE animals( 
+	id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
[OPTIONAL] you can mark the id column as primary key


    `PRIMARY KEY (id)`